migrations/1626119696313_update-asset-ids.ts

```diff
@@ -0,0 +1,30 @@
+import { MigrationBuilder } from 'node-pg-migrate'
+import { Asset } from '../src/Asset'
+
+const tableName = Asset.tableName
+const columns = ['id', 'asset_pack_id']
+
+export const up = (pgm: MigrationBuilder) => {
+	pgm.sql(`TRUNCATE TABLE ${tableName};`)
+
+	pgm.dropConstraint(tableName, 'assets_pkey')
+	pgm.addConstraint(tableName, 'assets_pkey', { primaryKey: 'id' })
+
+	// pgm won't infer this name correctly, so we need to nudge it a little
+	pgm.dropIndex(tableName, columns, {
+		name: 'assets_id_asset_pack_id_unique_index',
+	})
+	pgm.addIndex(tableName, 'id', { unique: true })
```

Comment:
Should we need update the `id` to be `uuid`?

Reply:
Good catch! I think so, yes